bundles/http/Citrix/Microsoft-Outlook/script.js

```diff
@@ -1,321 +1,733 @@
+    const end = moment().utc().add(30, 'd').format()
+    await calendarView(client, store, [actionParameters.userId], start, end)
+    const editRecurringEventReqeust = await client.fetch(`v1.0/me/events/${actionParameters.id}`, {
+        method: "PATCH",
+        body: JSON.stringify({
+            "subject": `${actionParameters.subject}`,
+            "body": {
+                "contentType": "HTML",
+                "content": `${actionParameters.content}`
+            },
+            "start": {
+                "dateTime": `${actionParameters.startDateTime}`,
+                "timeZone": `${actionParameters.timezone}`
+            },
+
+            "end": {
+                "dateTime": `${actionParameters.endDateTime}`,
+                "timeZone": `${actionParameters.timezone}`
+            },
+            "location": {
+                "displayName": `${actionParameters.location}`
+            },
+            "attendees": [{
+                "emailAddress": {
+                    "address": `${actionParameters.email1}`
+                },
+                "type": `${actionParameters.type1}`
+            },
+            {
+                "emailAddress": {
+                    "address": `${actionParameters.email2}`
+                },
+                "type": `${actionParameters.type2}`
+            },
+            {
+                "emailAddress": {
+                    "address": `${actionParameters.email3}`
+                },
+                "type": `${actionParameters.type3}`
+            },
+            {
+                "emailAddress": {
+                    "address": `${actionParameters.email4}`
+                },
+                "type": `${actionParameters.type4}`
+            },
+            {
+                "emailAddress": {
+                    "address": `${actionParameters.email5}`
+                },
+                "type": `${actionParameters.type5}`
+            },
+            {
+                "emailAddress": {
+                    "address": `${actionParameters.email6}`
+                },
+                "type": `${actionParameters.type6}`
+            }
+            ],
+            "allowNewTimeProposals": true,
+            "isOnlineMeeting": actionParameters.isOnlineMeeting,
+            "onlineMeetingProvider": `${actionParameters.onlineMeetingProvider}`,
+            "recurrence": {
+                "pattern": {
+                    "type": `${actionParameters.recurrencetype}`,
+                    "interval": 1,
+                    "daysOfWeek": [actionParameters.days],
+                    "dayOfMonth": actionParameters.dayOfMonth
+                },
+                "range": {
+                    "type": "endDate",
+                    "startDate": `${actionParameters.startdate}`,
+                    "endDate": `${actionParameters.enddate}`
+                }
+            }
+        })
+    })
+    if (!editRecurringEventReqeust.ok) {
+        throw new Error(editRecurringEventReqeust.text())
```

Comment:
What response does `editRecurringEventReqeust.text()` returns? Do you have any sample?

Reply:
It will return the response body in simple text format.

Reply:
I will keep this comment open for now. After seeing the sample response will close this comment.

Reply:
I have used console.log to get the response.

Reply:
Please provide some sample response once you have.

Reply:
```suggestion
        throw new Error(Edit Recurring Event with current timezone service action is failed ${editRecurringEventReqeust.status}:${editRecurringEventReqeust.statusText)
```

Modify the same for all the functions just by replacing service action names

Reply:
Still this change is not updated.